plugins/groq/tests/groq_test.ts

```diff
@@ -121,51 +63,35 @@ describe('toGroqRequestBody', () => {
       maxOutputTokens: 100,
       topP: 0.9,
       frequencyPenalty: 0.5,
-      logitBias: {
-        science: 12,
-        technology: 8,
-        politics: -5,
-        sports: 3,
-      },
+      logitBias: { science: 12, technology: 8, politics: -5, sports: 3 },
       seed: 42,
       topLogprobs: 10,
       user: 'exampleUser123',
-      custom: {
-        someCamelCase: 'someValue',
-      },
     },
   };
 
   it('should convert GenerateRequest to Groq request body', () => {
     const expectedOutput: ChatCompletionCreateParamsBase = {
-      messages: [
-        {
-          role: 'user',
-          content: 'Tell a joke about dogs.',
-        },
-      ],
-      model: 'llama3-8b-8192',
+      messages: [{ role: 'user', content: 'Tell a joke about dogs.' }],
+      model: 'llama3-8b-8192', // Match model name format in SUPPORTED_GROQ_MODELS
       temperature: 0.7,
       max_tokens: 100,
       top_p: 0.9,
       stop: ['\n'],
       frequency_penalty: 0.5,
-      logit_bias: {
-        science: 12,
-        technology: 8,
-        politics: -5,
-        sports: 3,
-      },
+      logit_bias: { science: 12, technology: 8, politics: -5, sports: 3 },
       seed: 42,
       top_logprobs: 10,
       user: 'exampleUser123',
-      response_format: {
-        type: 'text',
-      },
+      response_format: { type: 'text' },
     };
+
     const actualOutput = toGroqRequestBody('llama-3-8b', request);
     console.log(`actualOutput.stop: ${actualOutput.stop}`);
-    assert.deepStrictEqual(actualOutput, expectedOutput);
+    assert.deepStrictEqual(
```

Comment:
how come we have to add a JSON parse here? I dont think it's an issue, just wondering

Reply:
To remove undefined fields (I put a comment about it)